From: Bryan Donlan <bdonlan@gmail.com>
Date: Tue, 10 May 2011 16:12:38 -0400
Subject: [PATCH] Suppress help2man info-page reference

help2man refers users to an info page for full documentation by default;
since we have no info page, suppress this message
---
 ofxconnect/Makefile.am |    2 +-
 ofxdump/Makefile.am    |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ofxconnect/Makefile.am b/ofxconnect/Makefile.am
index 6c76924..4f1b402 100644
--- a/ofxconnect/Makefile.am
+++ b/ofxconnect/Makefile.am
@@ -25,4 +25,4 @@ TESTS = test-privateserver.sh
 
 ofxconnect.1: ofxconnect.cpp $(top_srcdir)/configure.in
 	$(MAKE) $(AM_MAKEFLAGS) ofxconnect$(EXEEXT)
-	help2man --output=ofxconnect.1 ./ofxconnect$(EXEEXT)
+	help2man -N --output=ofxconnect.1 ./ofxconnect$(EXEEXT)
diff --git a/ofxdump/Makefile.am b/ofxdump/Makefile.am
index bfcb105..a1d652f 100644
--- a/ofxdump/Makefile.am
+++ b/ofxdump/Makefile.am
@@ -16,7 +16,7 @@ endif
 
 ofxdump.1: ofxdump.cpp $(top_srcdir)/configure.in
 	$(MAKE) $(AM_MAKEFLAGS) ofxdump$(EXEEXT)
-	help2man --output=ofxdump.1 ./ofxdump$(EXEEXT)
+	help2man -N --output=ofxdump.1 ./ofxdump$(EXEEXT)
 
 MAINTAINERCLEANFILES  = cmdline.c cmdline.h
 EXTRA_DIST = cmdline.ggo
-- 
